src/status_im2/subs/subs_test.cljs

```diff
@@ -40,3 +42,24 @@
                 {"0x1" {:keycard-pairing "keycard-pairing-code"}}}
                {})]
       (is (= res "keycard-pairing-code")))))
+
+(def messages-state
+  [{:message-id "0x111" :album-id "abc" :albumize? true}
+   {:message-id "0x222" :album-id "abc" :albumize? true}
+   {:message-id "0x333" :album-id "abc" :albumize? true}
+   {:message-id "0x444" :album-id "abc" :albumize? true}
+   {:message-id "0x555" :album-id "efg" :albumize? true}])
+
+(def messages-albumized-state
+  [{:album        [{:message-id "0x444" :album-id "abc" :albumize? true}
+                   {:message-id "0x333" :album-id "abc" :albumize? true}
+                   {:message-id "0x222" :album-id "abc" :albumize? true}
+                   {:message-id "0x111" :album-id "abc" :albumize? true}]
+    :album-id     "abc"
+    :message-id   "abc"
+    :content-type constants/content-type-album}
+   {:message-id "0x555" :album-id "efg" :albumize? true}])
+
+(deftest find-albums
```

Comment:
@OmarBasem, these tests should be added to a new test file in `src/status_im2/subs/chat/messages_test.cljs`. This top level test file will be gone soon. I might even do it this week.

So the idea is to write one separate test file for every subscription file.

Reply:
@ilmotta Done!